### PR TITLE
Completed Java course Assignments

### PR DESCRIPTION
**Pull request by balnenikhil**
completed my Java Assignments, Please Review.

### CODE REVIEW

**Comment by Charul-Mehta** [comment]
Just a point to remember from next time:
I see there are 13 projects created, (1 for each assignment) which wasn't necessary. We can have different packages in the same project and still keep the assignments separate. Doing this will make your projects much lighter in weight as unnecessary project setup is not required.

**Review by Charul-Mehta — COMMENTED** [review]
I have checked the files and added few comments please check. Rest looks good to me

File: Assignments/JavaAssignment6/src/asssignment6/Constructor1.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package asssignment6;
+
+public class Constructor1 {
+    public Constructor1(int num){
```
[the remark this answers is not in the document]
Comment (balnenikhil) [reply Assignments/JavaAssignment6/src/asssignment6/Constructor1.java:4]: In Assignment It was mentioned to Call the another constructor inside one constructor.
Question : Create a class with two (overloaded) constructors. Using this, call the second constructor inside the first one

File: Assignments/JavaAssignment7/src/cyclefactory/CycleFactories.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+class Tricycle implements Cycle{
+    public void rideNow(){
+        System.out.println("Riding Tricycle");
+    }
+}
+class Trifactory implements CycleFactory{
+    public Cycle getCycle(){
+        return new Tricycle();
+    }
+}
```
[the remark this answers is not in the document]
Comment (balnenikhil) [reply Assignments/JavaAssignment7/src/cyclefactory/CycleFactories.java]: done the necessary changes.

File: Assignments/JavaAssignment10/src/linkedlist/LinkedList.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        System.out.println("The elements in List are:");
+        Node<T> temp = head;
+        do {
+            System.out.println(temp.data);
+            temp = temp.next;
+        } while (temp != null);
+
+    }
+}
+
```
[the remark this answers is not in the document]
Comment (balnenikhil) [reply Assignments/JavaAssignment10/src/linkedlist/LinkedList.java]: made the changes as suggested.

**Pull request merged**